scripts/check-spec.sh

```diff
@@ -32,7 +32,7 @@ check_spec() {
         printf "\n    Spec version is correct.\n"
     else
         printf "\n    Spec version should $version.\n"
-        exit 1
+        check_spec_result=1
```

Comment:
Better to use the upper case for variable in the shell.
```suggestion
        CHECK_SPEC_RESULT=1
```

Reply:
What's the difference?